lib/optimize/ModuleConcatenationPlugin.js

```diff
@@ -136,11 +141,11 @@ class ModuleConcatenationPlugin {
 					for(const warning of concatConfiguration.warnings) {
 						newModule.optimizationBailout.push((requestShortener) => {
 							const reason = getBailoutReason(warning[0], requestShortener);
-							const reasonPrefix = reason ? `: ${reason}` : "";
+							const reasonWithPrefix = reason ? ` (<- ${reason})` : "";
 							if(warning[0] === warning[1])
-								return `ModuleConcatenation: Cannot concat with ${warning[0].readableIdentifier(requestShortener)}${reasonPrefix}`;
+								return formatBailoutReason(`Cannot concat with unsupported ${warning[0].readableIdentifier(requestShortener)}${reasonWithPrefix}`);
```

Comment:
`unsupported`  could be confusing if two super supported ECMAScript modules can be concatenated because they are in different chunks.